README.md

```diff
@@ -4,6 +4,19 @@ Proxy that provides access to the VNC console of a Kubevirt VM.
 
 It can generate time limited tokens that are then used to access VNC.
 
+## Installation
+
+### Using SSP operator
+The [SSP operator](https://github.com/kubevirt/ssp-operator) can be configured to install VM Console Proxy together with
```

Comment:
`
SSP operator can be configured to install the VM Console Proxy together with a Route to expose it to the external network.
`

Reply:
done